docs: add Ajit Singh to the collaborators

ajitsinghkaler opened the pull request:
Ajit Singh is a newly added collborator after a few months of contributing add him to the contributors.json

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

gkalpak commented on aio/content/marketing/contributors.json:
```diff
@@ -576,6 +576,13 @@
+    "groups": ["Collaborators"],
+    "picture": "ajitsinghkaler0.jpg",
+    "twitter": "ajitsinghkaler",
+    "bio": "Software Engineer in Bangalore, India who loves to learn something new and is intrested in front end technologies."
```
```suggestion
    "bio": "Software Engineer in Bangalore, India who loves to learn something new and is interested in front end technologies."
```

gkalpak commented on aio/content/marketing/contributors.json:
```diff
@@ -576,6 +576,13 @@
     "picture": "cexbrayat.jpg",
     "bio": "Author of `Become a ninja with Angular (2+)` https://books.ninja-squad.com/angular - Angular trainer and @Ninja-Squad co-founder"
   },
+  "ajitsinghkaler0": {
```
Please use your GitHub username as key:

```suggestion
  "ajitsinghkaler": {
```

ajitsinghkaler committed:
docs: add Ajit Singh to the collaborators

Ajit Singh is a newly added collborator after a few months of contributing add him to the contributors.json

ajitsinghkaler committed:
fixup! docs: add Ajit Singh to the collaborators

gkalpak reviewed (approved):
:tada:
